server/pulp/server/webservices/urls.py

```diff
@@ -1,6 +1,13 @@
 from django.conf.urls import patterns, url
 
 from pulp.server.webservices.views.content import UploadSegmentResourceView
+from pulp.server.webservices.views.consumer_groups import (ConsumerGroupView,
```

Comment:
This should come before pulp.server.webservices.views.content for alphabetical imports